skills/golem-powers/context-audit/scripts/audit.sh

```
@@ -11,55 +11,74 @@ YELLOW='\033[0;33m'
 
 # ─────────────────────────────────────────────────────────────
-# 1. AVAILABLE CONTEXTS
+# 1. AUTO-LOADED RULES (.claude/rules/)
 # ─────────────────────────────────────────────────────────────
-echo -e "${CYAN}AVAILABLE CONTEXTS:${NC}"
+echo -e "${CYAN}AUTO-LOADED RULES (.claude/rules/):${NC}"
+
+if [ -d "$RULES_DIR" ]; then
+    RULE_COUNT=0
+    while IFS= read -r rule; do
+        name="${rule#$RULES_DIR/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)